ui/ui-config/__tests__/integration/happyPath.ts

```diff
@@ -92,16 +102,34 @@ describe("integration", () => {
   test(
     "it redirects to setup steps after clicking setup button",
     async () => {
-      await browser.get(`${env.url}/`);
+      await browser.get(`${url}/`);
       await browser.wait(until.elementLocated(by.className("btn-primary")));
       const button = browser.findElement(by.className("btn-primary"));
       await button.click();
 
       await browser.wait(until.elementLocated(by.id("setup")));
-      const url = await browser.getCurrentUrl();
-      expect(url).toMatch(/\/setup/);
-      await browser.get(url);
+      const currentUrl = await browser.getCurrentUrl();
+      expect(currentUrl).toMatch(/\/setup/);
+      await browser.get(currentUrl);
     },
     helper.mediumTime
   );
+
+  test("if all the stepup steps are complete, visiting / goes to profiles", async () => {
+    const setupSteps = await SetupStep.findAll();
+    for (const step of setupSteps) {
+      await step.update({ complete: true });
+    }
+
+    await browser.get(`${url}/`);
+    await browser.wait(until.elementLocated(by.className("btn-primary")));
+    const button = browser.findElement(by.className("btn-primary"));
+    await button.click();
+
+    await helper.sleep(1000);
+
+    const currentUrl = await browser.getCurrentUrl();
+    expect(currentUrl).toMatch(/\/profiles/);
+    await browser.get(currentUrl);
+  });
```

Comment:
@tealjulia @seancdavis - here's the test that started all of this! Now, we can update the SetupStep model, reload the page, and see that we are taken to `/profiles` rather than `/setup`